angular-workspace/projects/ni/nimble-angular/src/thirdparty/directives/ng_control.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copied from https://github.com/angular/angular/blob/e9b5dac9ecaa86914079a37176985c628c403aba/packages/forms/src/directives/ng_control.ts
```

Comment:
We can add our `[Nimble]` prefix to these comments as well

Reply:
Done